docs/book/src/security-guidelines/pod-security-standards.md

```diff
@@ -0,0 +1,211 @@
+# Pod Security Standards
+
+Pod Security Admission allows defining [Pod Security Standards] for creation of pods at the cluster level.
+
+The flavor `development-topology` for the docker provider used in [Quick Start](../user/quick-start.md) already includes a basic Pod Security Standard configuration.
+It is using variables and patches via ClusterClass to inject the configuration.
+
+## Adding a basic Pod Security Standards configuration to a ClusterClass
+
+By adding the following variables and patches Pod Security Standards can be added to every ClusterClass which references a [Kubeadm based control plane](../tasks/kubeadm-control-plane.md).
+
+### Adding the variables to a ClusterClass
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+spec:
+  variables:
+  - name: podSecurityStandard
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties: 
+          enabled: 
+            type: boolean
+            default: true
+            description: "enabled enables the patches to enable Pod Security Standard via AdmissionConfiguration."
+          enforce:
+            type: string
+            default: "baseline"
+            description: "enforce sets the level for the enforce PodSecurityConfiguration mode. One of privileged, baseline, restricted."
+            pattern: "privileged|baseline|restricted"
+          audit:
+            type: string
+            default: "restricted"
+            description: "audit sets the level for the audit PodSecurityConfiguration mode. One of privileged, baseline, restricted."
+            pattern: "privileged|baseline|restricted"
+          warn:
+            type: string
+            default: "restricted"
+            description: "warn sets the level for the warn PodSecurityConfiguration mode. One of privileged, baseline, restricted."
+            pattern: "privileged|baseline|restricted"
+  ...
+```
+
```

Comment:
Maybe worth adding a couple of notes:
- The version field in Pod Security Admission Config defaults to `latest` 
- The `kube-system` namespace is exempt from Pod Security Standards enforcement, because it runs control-plane pods that need higher privileges